packages/next/src/seo/utils.ts

```diff
@@ -0,0 +1,23 @@
+export interface Node {
+	description?: string | undefined | null;
+	excerpt?: string | undefined | null;
+	content?: string | undefined | null;
+}
+
+export const getNodeDescription = < T extends Node >(
```

Comment:
It's not the `NodeDescription` (thats the `description` field). Its either the `MetadataDescription` or alternativly go more colloquial with `getDescriptionFromNode()`